drivers/base/kdgdb/i386_sup.c

```diff
@@ -95,7 +95,7 @@ thread_to_reg(PETHREAD Thread, enum reg_name reg_name, unsigned short* size)
             case EIP:
                 KDDBGPRINT("Returning NULL for register %d.\n", reg_name);
                 *size = 4;
-                return &NullValue;
+                return (void*)&NullValue;
```

Comment:
You can as well change the function return type to be const. That would be more invasive though.